### DISCUSSION TITLE
Invoices API

### DISCUSSION THREAD

**Discussion by cianiandreadev** [kimai/kimai#2676] [General]
Hello everyone,

Sorry but I searched everywhere and I couldn't find a way to retrive the invoice list via REST apis.

Maybe this is being already answered somewhere but I didn't find it.

Why are not available any apis for the invoices? Would be nice to see at least the status and change through REST apis.

Thanks in advance.

**Comment by kevinpapst** [comment]
> Why are not available any apis for the invoices?

Because ... no one asked for this before.

**Reply by cianiandreadev** [reply]
Ah! I see 😄 
Ok. At least I stop searching.
